### PR TITLE
Add information how to create custom presets

### LINKED ISSUE #46: Add info to README about creating customized presets

### PR DESCRIPTION
**Pull request by jacekbogdanski**
Closes #46

### CODE REVIEW

**Comment by f1ames** [comment]
Rebased onto latest `master`.

**Review by f1ames — APPROVED** [review]
LGTM :+1: 

@godai78 could you please take a final look?

File: README.md [hunk 1]
```diff
@@ -36,3 +36,102 @@ Examples:
 
 	> # Build a "basic" release, including all plugins available.
 	> build.sh basic all
+
+## Creating Custom Presets
```
Comment (godai78) [inline README.md:40]:
```suggestion
## Creating custom Presets
```

File: README.md [hunk 2]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+
+Apart from using predefined presets, you can create a custom preset manually and use it with `build.sh` script. As an example, let's create a preset including basic configuration with [Editor Placeholder](https://ckeditor.com/cke4/addon/editorplaceholder) plugin.
```
Comment (godai78) [inline README.md:42]:
```suggestion
Apart from using predefined presets, you can create a custom preset manually and use it with the `build.sh` script. As an example, let's create a preset including basic configuration with the [Editor Placeholder](https://ckeditor.com/cke4/addon/editorplaceholder) plugin.
```

File: README.md [hunk 3]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+
+### 1. Adding build configuration file
+
+Create builder configuration file at `presets/custom-build-config.js`:
```
Comment (godai78) [inline README.md:46]:
```suggestion
Create a builder configuration file at `presets/custom-build-config.js`:
```

File: README.md [hunk 4]
```diff
@@ -36,3 +36,102 @@ Examples:
 
 	> # Build a "basic" release, including all plugins available.
 	> build.sh basic all
+
+## Creating Custom Presets
+
+Apart from using predefined presets, you can create a custom preset manually and use it with `build.sh` script. As an example, let's create a preset including basic configuration with [Editor Placeholder](https://ckeditor.com/cke4/addon/editorplaceholder) plugin.
+
+### 1. Adding build configuration file
```
Comment (godai78) [inline README.md:44]:
```suggestion
### 1. Adding the build configuration file
```

File: README.md [hunk 5]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+		link: 1,
+		editorplaceholder: 1,
+		toolbar: 1,
+		undo: 1,
+		wysiwygarea: 1
+	}
+};
+```
+
+### 2. Adding editor configuration file
```
Comment (godai78) [inline README.md:97]:
```suggestion
### 2. Adding the editor configuration file
```

File: README.md [hunk 6]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+
+Create editor configuration file at `presets/custom-ckeditor-config.js`:
```
Comment (godai78) [inline README.md:99]:
```suggestion
Create an editor configuration file at `presets/custom-ckeditor-config.js`:
```

File: README.md [hunk 7]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+		{ name: 'links' },
+		{ name: 'insert' },
+		{ name: 'styles' },
+		{ name: 'colors' },
+		{ name: 'tools' },
+		{ name: 'others' },
+		{ name: 'about' }
+	];
+
+	// The default plugins included in the basic setup define some buttons that
```
Comment (godai78) [inline README.md:120]:
```suggestion
	// The default plugins included in the basic setup define some toolbar items that
```

File: README.md [hunk 8]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+	// are not needed in a basic editor. They are removed here.
```
Comment (godai78) [inline README.md:121]:
```suggestion
	// are not needed in a basic editor. These are removed here.
```

File: README.md [hunk 9]
```diff
@@ -36,3 +36,102 @@ Examples:
...
+
+	// Dialog windows are also simplified.
+	config.removeDialogTabs = 'link:advanced';
+
+	// Finally, configure editor placeholder text.
+	config.editorplaceholder = 'Type something here...';
+};
+```
+
+### 3. Building custom preset
```
Comment (godai78) [inline README.md:132]:
```suggestion
### 3. Building a custom preset
```

**Review by godai78 — COMMENTED** [review]
Minor fixes.

File: README.md (hunk 7 above)
Comment (jacekbogdanski) [reply README.md:120]: Maybe `toolbar buttons` as everything in a toolbar is a button?

**Comment by f1ames** [comment]
Rebased onto latest `master`.

**Review by f1ames — APPROVED** [review]
LGTM :+1:

**Pull request merged**